contracts/adapters/Managing.sol

```diff
@@ -50,19 +50,6 @@ contract ManagingContract is IManaging, DaoConstants, MemberGuard {
         revert("fallback revert");
     }
 
-    /**
-     * @notice returns the voting adapter name that is configured in the DAO.
-     */
-    function getVotingAdapterName(DaoRegistry dao)
-        external
-        view
-        override
-        returns (string memory)
-    {
-        IVoting votingContract = IVoting(dao.getAdapterAddress(VOTING));
-        return votingContract.getAdapterName();
-    }
-
```

Comment:
@adridadou how can we load the current voting contract name from the client side without this function? Is there an alternative for that?

Reply:
you call the voting contract directly. No need for it there.
We have the address of the voting contract and therefore we can call votingContract.getAdapterName() from the UI